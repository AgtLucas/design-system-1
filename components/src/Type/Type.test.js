import { Type } from './';

describe('Type', () => {
  it('matches snapshot', () => {
    expect(Type).toBeDefined();
    expect(Type).toMatchSnapshot();
  });
});

Object.keys(Type).forEach(name => (
  describe(`Type.${name}`, () => {
    it('matches snapshot', () => {
      expect(Type[name]).toBeDefined();
      expect(Type[name]).toMatchSnapshot();
    });
  })
));
