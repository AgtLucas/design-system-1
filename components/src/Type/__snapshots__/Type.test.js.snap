// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Type matches snapshot 1`] = `
Object {
  "Text": [Function],
}
`;

exports[`Type.Text matches snapshot 1`] = `[Function]`;
